sdk/python/kfp/cli/cli.py

```diff
@@ -85,6 +85,9 @@ def _install_completion(shell: str) -> None:
 @click.option(
     '--other-client-secret',
     help=parsing.get_param_descr(client.Client, 'other_client_secret'))
+@click.option(
+    '--existing-token',
+    help=parsing.get_param_descr(client.Client, 'existing_token'))
```

Comment:
Thanks @tumido can you also add this change to: https://github.com/kubeflow/pipelines/blob/master/sdk/RELEASE.md#features ?